catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -123,9 +124,16 @@ public void init() {
             IOUtils.copy(attachmentInfo.getStream(), temporaryFileBackedOutputStream);
 
             for (Splitter splitter : lookupSplitters(attachmentInfo.getContentType())) {
-              if (attemptToSplitAndStore(
-                  response, listType, attachmentInfo, temporaryFileBackedOutputStream, splitter))
-                break;
+              try {
+                if (attemptToSplitAndStore(
+                    response, listType, attachmentInfo, temporaryFileBackedOutputStream, splitter))
+                  break;
+              } catch (StopSplitterExecutionException e) {
```

Comment:
❗️ Although caught here and documented as a throws clause in other methods, I do not see anywhere it would actually be thrown.

Reply:
Is it a requirement that it actually be thrown?  This is simply enabling the option to do so.

Reply:
ah! That was not clear